src/main/kotlin/frc/robot/subsystems/climb/ClimbIOTalonFX.kt

```diff
@@ -0,0 +1,56 @@
+package frc.robot.subsystems.climb
+
+import com.ctre.phoenix.motorcontrol.NeutralMode
+import com.ctre.phoenix.motorcontrol.TalonSRXControlMode
+import com.ctre.phoenix6.controls.StrictFollower
+import com.ctre.phoenix6.hardware.TalonFX
+import com.ctre.phoenix.motorcontrol.can.TalonSRX
+import com.ctre.phoenix6.controls.DutyCycleOut
+import edu.wpi.first.units.Units
+import frc.robot.Ports
+
+class ClimbIOTalonFX : ClimbIO {
+    override val inputs = LoggedClimbInputs()
+    private val mainMotor = TalonFX(Ports.Climb.MAIN_MOTOR_ID)
+    private val auxMotor = TalonFX(Ports.Climb.AUX_MOTOR_ID)
+    private val stopperMotor = TalonSRX(Ports.Climb.STOPPER_ID)
+
+    private val percentOutput = DutyCycleOut(0.0).withEnableFOC(true)
+
+    init {
+        mainMotor.configurator.apply(ClimbConstants.MOTOR_CONFIG)
+        auxMotor.configurator.apply(ClimbConstants.MOTOR_CONFIG)
+        auxMotor.setControl(StrictFollower(mainMotor.deviceID))
+
+        stopperMotor.configFactoryDefault()
+        stopperMotor.enableCurrentLimit(true)
+        stopperMotor.enableVoltageCompensation(true)
+        stopperMotor.configVoltageCompSaturation(12.0)
```

Comment:
Be consistent: if some values are constants, all should be constants (and vice versa).

Reply:
I don't think values that are used once in a single place should be stored in a separate file.

Reply:
I agree with the compensation saturation being a constant but There is no situation that we'll want to turn off current limits or voltage compensation. I really don't see a reason to make them a constant.

Reply:
I'm not saying they should be constants (I usually think it's better to not use constants in these cases), but consistency is more important. Let's leave it for now :)